code/modules/clothing/suits/jobs.dm

```diff
@@ -216,6 +216,9 @@
 	inhand_icon_state = null
 	blood_overlay_type = "coat"
 	body_parts_covered = CHEST|ARMS
+	allowed = list(
+		/obj/item/stamp
+	)
```

Comment:
This'll make it so these coats can't hold anything else a coat can normally hold, like PDAs or flashlights

Reply:
I tested it in a localhost and it could still hold PDAs. Same to assume it can hold flashlights too